WalletLibrary/WalletLibrary/Networking/Entities/OpenID4VCI/CredentialConfiguration.swift

```diff
@@ -42,6 +42,53 @@ struct CredentialDefinition: Decodable
     let display: [LocalizedDisplayDefinition]?
 }
 
+extension CredentialConfiguration
+{
+    /// Get `VerifiedIdStyle` from metadata in preferred locale.
+    func getLocalizedVerifiedIdStyle(withIssuerName issuerName: String, mapper: any Mapping) -> VerifiedIdStyle
+    {
+        let definition = getPreferredLocalizedDisplayDefinition()
+        let logo = definition?.logo.flatMap { try? mapper.map($0) }
+        
+        let style = BasicVerifiedIdStyle(name: definition?.name ?? "",
+                                         issuer: issuerName,
+                                         backgroundColor: definition?.background_color ?? "",
+                                         textColor: definition?.text_color ?? "",
+                                         description: definition?.description ?? "",
+                                         logo: logo)
+        
+        return style
+    }
+    
+    private func getPreferredLocalizedDisplayDefinition() -> LocalizedDisplayDefinition?
+    {
+        guard let displayDefinitions = self.credential_definition?.display else
+        {
+            return nil
+        }
+        
+        if displayDefinitions.count == 1
+        {
+            return displayDefinitions.first
+        }
+        
+        let preferredLanguages = Locale.preferredLanguages
+        
+        for language in preferredLanguages 
+        {
+            for definition in displayDefinitions 
+            {
+                if definition.locale == language
+                {
+                    return definition
+                }
+            }
+        }
+        
+        return nil
```

Comment:
Could we change the order of these checks so we try optimistically for the preferred language first, then fall back to the first defined language, and at the worst case (no definitions found) return nil?

Reply:
sure!